tests/integration/config/plugins/integration-tests/data/tests/tests_common.txt

```diff
@@ -539,6 +539,118 @@ test "Load First Savegame"
 			key l
 
 
+test "Create New Pilot"
+	status partial
+	description "Helper function to create a new pilot."
+	sequence
+		input
+			key l
+		# new pilot
+		input
+			key n
+		# start
+		input
+			key s
+		# first name: a
+		input
+			key a
+		input
+			key "return"
+		# last name: b
+		input
+			key b
+		input
+			key "return"
+		# skip text
+		input
+			key "return"
+		# go to the main landing panel
+		input
+			key l
+
+
+test "Buy First Ship"
+	status partial
+	description "Helper function to buy the first ship of the shipyard."
+	sequence
+		input
+			key s
+		# buy first ship
+		input
+			key "down"
+		input
+			key b
+		# confirm
+		input
+			key "return"
+		# back to landing
+		input
+			key l
+
+
+test "Wait 55 steps"
+	status partial
+	sequence
+		apply
+			"test: wait 55" = 55
+		label wait_load
+		apply
+			"test: wait 55" -= 1
+		input
+		branch wait_load
+			"test: wait 55" > 0
+
+
+test "Wait 10 steps"
+	status partial
+	sequence
+		apply
+			"test: wait 10" = 10
+		label wait
+		apply
+			"test: wait 10" -= 1
+		input
+		branch wait
+			"test: wait 10" > 0
+
+
+test "Start Game"
+	status active
+	description "Test if one can start the game correctly: choose a name and buy a ship."
+	sequence
+		watchdog 1000
+		call "Create New Pilot"
+		# the game needs a few steps to load the new pilot
+		call "Wait 55 steps"
+		call "Buy First Ship"
+		# game needs time to process showing the mission
+		call "Wait 10 steps"
+		# beat it, old timer
+		input
+			key 3
+		input
+			key "return"
+		# to be sure
+		input
+			key l
+		# normally the mission being launched means the conditions are refreshed (or they always are?) and we don't need to launch
+		# but to be sure:
+		call "Depart"
+		# save these in temporary conditions to know what they are if it fails, for easier debugging
+		apply
+			"test: name: a b" = "name: a b"
+			"test: Intro [0]: offered" = "Intro [0]: offered"
+			"test: Intro [0]: declined" = "Intro [0]: declined"
+			"test: total ships" = "total ships"
+			"test: ship model: Shuttle" = "ship model: Shuttle"
+			"test: ship model: Sparrow" = "ship model: Sparrow"
+			"test: ship model: Star Barge" = "ship model: Starg Barge"
+		assert
+			"name: a b" == 1
+			"total ships" == 1
+			# not checking for the intro because it shouldnt fail if it isnt offered anymore or changes name
```

Comment:
Why bother storing the value, then?
Also, there's too many pronouns here:
```suggestion
			# Not checking for the intro mission conditions because the test shouldn't fail if the mission isn't offered or its name changes.
```

Reply:
well may as well store it to know what happens in case this fails